app/_redirects

```diff
@@ -671,12 +675,12 @@
 
 # GENERAL REDIRECTS TO /GATEWAY/
 /enterprise/changelog      /gateway/changelog
-/enterprise/2.6.x/introduction/key-concepts     /konnect-platform/key-concepts
-/enterprise/2.7.x/introduction/key-concepts     /konnect-platform/key-concepts
-/enterprise/latest/introduction/key-concepts    /konnect-platform/key-concepts
-/enterprise/2.6.x/introduction/support-policy   /konnect-platform/support-policy
-/enterprise/2.7.x/introduction/support-policy   /konnect-platform/support-policy
-/enterprise/latest/introduction/support-policy  /konnect-platform/support-policy
+/enterprise/2.6.x/introduction/key-concepts     /gateway/latest/availability-stages/
```

Comment:
The URL redirected to the right page, the page just results in a 404.

Reply:
Yep, they're all 404s for now, this PR was originally targeting `main` so the content isn't in 3.0 yet.